Composer/packages/server/src/models/settings/defaultSettingManager.ts

```diff
@@ -37,6 +43,7 @@ export class DefaultSettingManager extends FileSettingManager {
       },
       telemetry: {
         logPersonalInformation: false,
+        logActivities: false,
```

Comment:
As per comment above, this should be set to true by default.

Reply:
fixed